_data/fellows.yml

```diff
@@ -1,15 +1,7 @@
-- name: Will
```

Comment:
This is also not required to be changed. It will also lead to a bunch of conflicts. Please fetch upstream before pushing to your fork.